dev/com.ibm.websphere.appserver.features/visibility/private/io.openliberty.springBootHandler-4.0.feature

```diff
@@ -0,0 +1,19 @@
+-include= ~${workspace}/cnf/resources/bnd/feature.props
+symbolicName=io.openliberty.springBootHandler-4.0
+WLP-DisableAllFeatures-OnConflict: true
+visibility=private
+-features=io.openliberty.jakartaeePlatform-11.0
```

Comment:
Why isn't this `jakartaeePlatform-10.0`?

Reply:
If you leave it as EE11 then surely you won't be able to GA unless EE11 GAs first? I don't know what the exact timing will be like?

Reply:
done, thanks!